### PR TITLE
glib: add version 2.69.0

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **glib/2.69.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

File: recipes/glib/all/conandata.yml [hunk 1]
```diff
@@ -1,55 +1,58 @@
 sources:
-  "2.65.0":
```
Comment (uilianries) [inline recipes/glib/all/conandata.yml:2]: Don't need to comment versions in conandata.yml. The CI bot only reads config.yml to decide what will be built.

File: recipes/glib/config.yml [hunk 2]
```diff
@@ -1,37 +1,39 @@
 versions:
-  "2.65.0":
```
Comment (uilianries) [inline recipes/glib/config.yml:2]: Remove instead of commenting

File: recipes/glib/all/conandata.yml (hunk 1 above)
Comment (ericLemanissier) [reply recipes/glib/all/conandata.yml:2]: I thought there was a hook checking precisely this, to avoid forgetting ?

Comment (uilianries) [reply recipes/glib/all/conandata.yml:2]: True! Hook 52 to be more exact.

**Review by uilianries — APPROVED** [review]

**Comment by ericLemanissier** [comment]
@jgsogo it seems qt is not the only package giving a hard time to C3I

**Comment by jgsogo** [comment]
> @jgsogo it seems qt is not the only package giving a hard time to C3I

I'm with this one as well, it is already building (for 3 hours now)

**Review by theirix — APPROVED** [review]

**Pull request merged**